index.html

```diff
@@ -65,7 +65,131 @@ <h2 class="main-title main-section-subtitle">I'm a software<br> developer</h2>
+            </div>
+        </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
+            </div>
+        </div>
+    </section>
+    <!--end of work section-->
+
+    <!--about me section-->
+    <section class="aboutme-container">
+        <img class="frame1" src="assets/images/Frame1.png" alt="frame 1">
+        <img class="frame2" src="assets/images/Frame2.png" alt="frame 2">
+        <div class="aboutme-text-area ">
+                <h1  class="aboutme-title">About me</h1>
+            <p class="aboutme-paragraph">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                &#39;t hestiate to contact me.
+            </p>            
+            <a><div class="btn-get-resume">Get My Resume</div></a>
+        </div>
+        <div class="aboutme-cards-area">
+            <div class="aboutme-card-item">
+                <div class="aboutme-card-logo">
+                    <img class="aboutme-logo" src="assets/icons/aboutme-icon1.png" alt="aboutme logo 1">
+                </div>
+                <div class="aboutme-card-details">
+                    <h2 class="aboutme-card-title">Languages</h2>
+                    <div class="aboutme-tags-container">
+                        <div class="aboutme-tags-item">JavaScipt</div>
+                        <div class="aboutme-tags-item">Ruby</div>
+                        <div class="aboutme-tags-item">Html</div>
+                        <div class="aboutme-tags-item">css</div>
+                    </div>
+                </div>
+            </div>
+            <div class="aboutme-card-item">
+                <div class="aboutme-card-logo">
+                    <img class="aboutme-logo" src="assets/icons/aboutme-icon2.png" alt="about me logo 2">
+                </div>
+                <div class="aboutme-card-details">
+                    <h2 class="aboutme-card-title">Frameworks</h2>
+                    <div class="aboutme-tags-container">
+                        <div class="aboutme-tags-item">Bootstrap</div>
+                        <div class="aboutme-tags-item">Ruby on rails</div>
+                        <div class="aboutme-tags-item">RSpec</div>
+                        <div class="aboutme-tags-item">Capybara</div>
+                        <div class="aboutme-tags-item">Selenium</div>
+                    </div>
```

Comment:
- [ ] As I mentioned earlier, one of the purposes of this template is to use `ul`'s and `li`'s items, so please change this `div`'s to make then lists. Note that your final result should be a nested list, similar to this:
```
<ul class="main-container">
  <li class="first-item"></li>
    <ul class="second-container">
      <li class="first-item-second-container"></li>
      <li class="second-item-second-container"></li>
      <li class="third-item-second-container"></li>
    </ul>
  <li class="second-item"></li>
  <li class="third-item"></li>
</ul>
```
So please apply the necessary changes.